src/components/MatchCard.tsx

```diff
@@ -14,7 +14,7 @@ export const MatchCard = ({
 }) => {
   if (mini) {
     return (
-      <Card p={'1em'} w={'10em'} textAlign={'center'}>
```

Comment:
This is breaking scaling
![Screenshot 2022-11-23 at 00 11 13](https://user-images.githubusercontent.com/6239715/203445969-a1ac178c-48a1-40cd-9045-4506e8e06ad8.png)
Everything else looks good

Reply:
Nvm that was already broken but also  I kinda like the consistent icons vs full width, a day with few matches will causes cards to be too fat

Reply:
I mean looks fine to me if this is the semi-finals, can set a maxW on the card instead, wdyt ?
![image](https://user-images.githubusercontent.com/58405138/203449809-aa1262c6-3084-461f-929f-1fb1735784f6.png)